src/device/rp/rp2040-plus.go

```diff
@@ -0,0 +1,51 @@
+// Hand created file. DO NOT DELETE.
+// Definitions that are missing in src/device/rp/rp2040.go generated from SVDs
+
```

Comment:
This comment seems redundant: everything not marked generated is not, and I'm not even sure the definitions are missing per se. Are DREQ constants in scope for SVD?

Also, I believe a consistent file name would use underscores. `plus` is not that informative. Perhaps `rp2350_consts.go` for now?

Reply:
For naming and comments, I tried to follow the example provided by @deadprogram above.
About the definitions, I was unable to spot them in the generated files, and also did a brief search in SVDs.